docs/UserGuide.adoc

```diff
@@ -162,25 +162,33 @@ recipe to contain two instructions:
     * boil eggs
     * slice apples
 
-=== List Recipes — `list`
-Lists out all the recipes managed by *CookBuddy*.
+=== Recipe Listing — `list`
+You can list all the existing recipes from *CookBuddy* using the `list` command.
 
-=== Find Recipes — `find`
-Find recipes with a particular parameter.
+Format: `list`
+
+=== Recipe Finding — `find`
+You can find an existing recipe with a particular parameter from *CookBuddy* using the `find` command.
+
+Format: `find [n/NAME [...]] [ing/INGREDIENT [...]]
 
 [NOTE]
 *CookBuddy* can find recipes from one parameter at a time.
 
-==== Find by name - `find [n/NAME [...]]`
-Example: `find n/sandwich`
-Finds recipes that contain the word `sandwich` in their name.
+==== Finding a recipe by name
+You can find a recipe by its name by appending `n/NAME [...]` to a `find` command.
+
+Example: You can type `find n/sandwich` and *CookBuddy* will display the recipes named sandwich to you.
 
-==== Find by ingredient - `find [ing/INGREDIENT [...]]`
-Example: `find ing/bread`
-Finds recipes that contain the word `bread` in their ingredients.
+==== Finding a recipe by ingredient
+You can find a recipe by its ingredient by appending `[ing/INGREDIENT [...]]` to a `find` command.
 
-=== Count Recipes — `count`
-Displays the total number of recipes stored in *CookBuddy*.
+Example: You can type `find ing/bread` and *CookBuddy* will display the recipes containing bread as an ingredient to you.
+
+=== Recipe counting — `count`
```

Comment:
```suggestion
=== Counting recipes — `count`
```
let's keep the naming conventions consistent for now. If we agree to phrase it differently later, then we'll open a new PR.